lib/config/config_test.go

```diff
@@ -173,33 +174,27 @@ func TestDeviceConfig(t *testing.T) {
 
 		cfg := wr.Wrapper.(*wrapper).cfg
 
-		expectedFolders := []FolderConfiguration{
-			{
-				ID:               "test",
-				FilesystemType:   FilesystemTypeBasic,
-				Path:             "testdata",
-				Devices:          []FolderDeviceConfiguration{{DeviceID: device1}, {DeviceID: device4}},
-				Type:             FolderTypeSendOnly,
-				RescanIntervalS:  600,
-				FSWatcherEnabled: false,
-				FSWatcherDelayS:  10,
-				Copiers:          0,
-				Hashers:          0,
-				AutoNormalize:    true,
-				MinDiskFree:      Size{1, "%"},
-				MaxConflicts:     -1,
-				Versioning: VersioningConfiguration{
-					Params: map[string]string{},
-				},
-				WeakHashThresholdPct: 25,
-				MarkerName:           DefaultMarkerName,
-				JunctionsAsDirs:      true,
-				MaxConcurrentWrites:  maxConcurrentWritesDefault,
-				XattrFilter: XattrFilter{
-					Entries: []XattrFilterEntry{},
-				},
-			},
-		}
+		fc := FolderConfiguration{}
+		structutil.SetDefaults(&fc)
```

Comment:
We don't want to do this here. The purpose of the test, such as it is, is to verify that folders are created with the default values we expect. Running the default-setter on the expected result makes it an `a == a` type of comparison, so I think we want to keep this manually defined.